ako-operator/controllers/configmap_test.go

```diff
@@ -61,9 +61,20 @@ var cmJson = `
 		"nodeValue": "value",
 		"passhtroughShardSize": "SMALL",
 		"serviceEngineGroupName": "test-group",
-		"serviceType": "ClusterIP",
+		"serviceType": "NodePort",
```

Comment:
I have changed serviceType to **NodePort**, because **nodeKey** and **nodeValue** are set in the Configmap. Either the serviceType had to be changed to NodePort or nodeKey and nodeValue had to be removed.

Reply:
The serviceType field has been changed to default **ClusterIP**, and the **nodeKey** and **nodeValue** fields have been removed.